modules/security-context-constraints-pre-allocated-values.adoc

```diff
@@ -4,7 +4,7 @@
 
 ifdef::openshift-origin,openshift-enterprise,openshift-dedicated[]
 [id="security-context-constraints-pre-allocated-values_{context}"]
-= About pre-allocated Security Context Constraints values 
+= About pre-allocated security context constraints values 
 
 The admission controller is aware of certain conditions in the Security Context
 Constraints (SCCs) that trigger it to look up pre-allocated values from a namespace and
```

Comment:
This one that spans lines 9-10 might have missed your search because it's split?

Reply:
You're right. One of the reasons that we stopped the hard wraps :)